docs/15-deploy-to-localstack.md

```diff
@@ -55,6 +56,22 @@ instance, as well as running partition and file status reports.
 ./scripts/utility/adminClient.sh <instance-id>
 ```
 
+## Standard ingest
+
+To ingest some data into the `system-test` table in your instance, you can run the following script in the
+`scripts/deploy/localstack` folder.
+
+```shell
+./ingestFiles <instance-id> <file1.parquet> <file2.parquet> <file3.parquet> ....
+```
+
+This script will upload the provided files to an ingest source bucket in LocalStack, create ingest jobs, and
+send them to the ingest job queue. It will then build the ingest-runner docker image, and launch a container for it,
+which will take the ingest job off the queue and perform the ingest.
```

Comment:
Can we add a way to generate some data locally? You can raise that as a separate issue if you prefer.

Reply:
Sure, raised this: https://github.com/gchq/sleeper/issues/1151